XenModel/BrandManager.cs

```diff
@@ -52,6 +53,7 @@ static BrandManager()
             ProductVersionPost82 = customBranding.ProductVersionText;
             XcUpdatesUrl = customBranding.XcUpdatesUrl;
             CfuUrl = customBranding.CfuUrl;
+            SourceUrl = XcUpdatesUrl.Replace("XCUpdates.xml", "XenCenter-source.zip");

```

Comment:
We're not using this value in the `BrandManager`. Should we remove it?

Reply:
probably